portfolio/src/main/webapp/style.css

```diff
@@ -1,78 +1,198 @@
-*{margin:0px; padding:0px;}
+* {
+  margin: 0px;
+  padding: 0px;
+}
+
 /*
-Purpose of this line it allow the background to extend throughout the whole page
-When only applied this padding rule to the body there was still an issue with the margin around the top by default
+Purpose of this line it allows the background to extend throughout the whole page
+When only applied this padding rule to the body there was still an issue with
+the margin around the top by default.
 */
+
 #content {
   width: 100%;
-  height:100vh;
-  /*background-color:orange;*/
+  height: 100vh;
   background-image: url("images/websiteBackground.png");
 }
 
 #greeting-container {
   margin-top: 20px;
 }
-#midTile {
-  position:relative;
-  text-align:center;
-  left:7.5%;
-  width:85%;
-  height:95vh;
-  background-color:teal;
-  opacity:.6
+
+#mid-tile {
+  position: relative;
+  text-align: center;
+  left: 7.5%;
+  width: 85%;
+  height: 95vh;
+  background-color: rgba(0, 128, 128, .6);
+}
+
+#icons {
+  padding-top: .5%;
+}
+
+#headshot {
+  width: 100%;
+  height: 100%;
+}
+
+.front-tile-logo {
+  width: 40%;
+  height: 40%;
 }
 
-/*Should find some generalizable way to autofit text in this box*/
+/*We should find some generalizable way to autofit text in this box.*/
+
 .tile {
-  float:left;
-  width: 31%;
-  height: 31%;
-  padding:1%;
-  background-clip:content-box; 
-  /*this handy line makes sure that padding is not 
-  also colored by the background color*/ 
-  background-color:yellow; 
+  /*Style was changed to have .33% at end so elements are perfectly centered
+  with that value of padding.*/
+  float: left;
+  width: 31.33%;
+  height: 31.33%;
+  padding: 1%;
+  vertical-align: middle;
+  background-clip: content-box;
+  /*This handy line makes sure that padding is not also colored by the
+  background color.*/
+  background-color: rgba(255, 255, 0, .65);
+}
 
+.portrait-tile-back {
+  overflow: auto;
+  width: 100%;
+  height: 100%
+}
+
+.tile-back img, .tile-front img, .portrait-tile-back img {
+  width: 100%;
+  height: 100%;
+  /*Uncomment line if don't want to distort image resolution.*/
+  /*object-fit:cover;*/
```

Comment:
Prefer not to submit commented-out code. If it's really necessary to be able to change something easily, it can be a flag or parameter (obviously not quite relevant to CSS). If not, it probably doesn't need to be in the codebase. Commented-out code tends to get out of sync with the rest of the code.